src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,11 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService userService = new UserService();
+        User user = userService.findByEmail(email);
+        if (user != null) {
```

Comment:
you can use ternary operator here

Reply:
i tried something like this
return (this.userService.findByEmail(email) != null) ? this.userService.findByEmail(email).getPassword().equals(password) : false;
but stylecheck scolded me because of to long code line (((